backend/src/test/java/de/neuefische/backend/service/RecipeServiceTest.java

```diff
@@ -18,21 +21,56 @@ class RecipeServiceTest {
     @Test
     void getAllRecipes() {
         //GIVEN
-        Recipe recipe = new Recipe("1", 30,"Erdbeerkuchen","2 Eier"
-                ,"gramm",200,"backen","leckeres Rezept"
-                ,"Kristina","https://www.","Winter");
-        List<Recipe> expected = new ArrayList<>(List.of(recipe));
+        Recipe recipe = new Recipe("1", "30", "Erdbeerkuchen",
+                List.of(new Ingredients("Ente", "1", null)),
+                List.of(new Method("backen")),
+                "leckeres Rezept", "Kristina", "",List.of(new Categories("Sommer,Winter")));
 
         //WHEN
-        when(recipeRepository.findAll()).thenReturn(expected);
+        when(recipeRepository.findAll()).thenReturn(Collections.singletonList(recipe));
```

Comment:
Macht das Sinn, dass du bei einer getAllRecipes Methode ein singletonList zurück bekommst?
In deinem Test hast du nur ein Rezept angelegt, aber ich glaube deinen Test würde fehlschlagen, wenn du zwei Test  Rezepte auf deiner Liste hättest.

Reply:
meine Test schlagen bereits länger fehl seitdem ich die Datenstruktur geändert habe und meine Array's nicht mehr akzeptiert werden. Alle Test's schauhe ich mir grad an und wie das da hinkommt... keine Ahnung :)